src/pages/legal-disclamer.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```

Comment:
shouldn't the file be .jsx?

Reply:
Same as for hez-circ-supply.js page. It has been changed.